src/fasttext.cc

```diff
@@ -239,18 +239,21 @@ void FastText::loadModel(std::istream& in) {
 }
 
 void FastText::printInfo(real progress, real loss, std::ostream& log_stream) {
-  // clock_t might also only be 32bits wide on some systems
-  double t = double(clock() - start_) / double(CLOCKS_PER_SEC);
+  std::chrono::steady_clock::time_point end = std::chrono::steady_clock::now();
+  double t = std::chrono::duration_cast<std::chrono::seconds> (end - start_).count();
```

Comment:
Please cast to `std::chrono::duration<double>` instead of `std::chrono::seconds`. Otherwise, the duration will be floored to an integer, before being casted to a double.

Reply:
Thank you for your review! I believe I have fixed this now. 
Many thanks,
Dom